### PR TITLE
fix: add missing deps for Button

### PR DESCRIPTION
**Pull request by ghoshnirmalya**
## Description
This PR adds the missing dependency from the Button component.

## Screenshot
|Screenshot of the issue|
|-|
|Screenshot 2023-05-01 at 12 18 45 PM|

## Solution
Adding the following missing dependency fixes the issue:
```diff
import * as React from "react"
+ import { Slot } from "@radix-ui/react-slot"
import { VariantProps, cva } from "class-variance-authority"
```

### CODE REVIEW

File: apps/www/content/docs/components/button.mdx [hunk 1]
```diff
@@ -20,7 +20,13 @@ npx shadcn-ui add button
     <AccordionTrigger>Manual Installation</AccordionTrigger>
     <AccordionContent>
 
-1. Copy and paste the following code into your project.
+1. Install the `@radix-ui/react-slot` component from radix-ui:
+
+```bash
+npm install @radix-ui/react-aspect-ratio
```
Comment (its-monotype) [inline apps/www/content/docs/components/button.mdx:26]:
```suggestion
npm install @radix-ui/react-slot
```

**Review by its-monotype — APPROVED** [review]

**Comment by its-monotype** [comment]
@ghoshnirmalya A bit off-top, but what is this font in your IDE, it looks awesome

**Comment by ghoshnirmalya** [comment]
@its-monotype Thanks. The font is [Fira Code](https://github.com/tonsky/FiraCode).

**Comment by SamHep0803** [comment]
Could this dependency also be added to the CLI command?

**Review by shadcn — APPROVED** [review]
Thank you. Sorry it took so long to review. Been crazy lately!

**Pull request merged**